javascript/latexContent.js

```diff
@@ -148,6 +148,7 @@ export const preamble = `\\documentclass[a4paper, 12pt]{book}
 }
 
 \\newcommand{\\OptionalPar}[2][]{\\ensuremath{\\text{\\textrm{\\sl #2}}_{#1}}}
+\\mathchardef\\mhyphen="2D 
```

Comment:
Would it make sense with a comment here in the LaTeX so we remember what this means in the future?

Reply:
comment added